tox.ini

```diff
@@ -23,6 +23,7 @@ setenv = MY_CONFIG = nonexistent
 commands =
     pip install -e .[testing]
     pip install cachew
+    pip install orjson
```

Comment:
> I think the basic solution here would be to create a separate tests/serialize.py file and call that in testenv:tests-all?

Yeah, I guess it's good to start with... Basically in principle this belongs to `tests-core`, for `tests-all` I had in mind that it runs the 'module tests'. But also want to have a way of running tests with/without optional libraries (like `pandas`/`cachew` and `orjson`) to make sure the fallbacks work properly.
Maybe something like a parameterized test which would temporary 'shadow' optional modules from `sys.modules` could work... Or a separate tier of tests which runs core + optional libraries. Not sure, but okay to keep it this way for now I guess.

Reply:
> But also want to have a way of running tests with/without optional libraries (like pandas/cachew and orjson) to make sure the fallbacks work properly

Ah, thats true, I don't think I'm currently testing for that. Would require to have a test somewhere in `tests-core` (perhaps adding it directly to the  that runs before `orjson` is installed, and then different tests for when orjson is installed. Perhaps could add another test which tests the fallback and add a regular (non-starred) import in [here](https://github.com/karlicoss/HPI/blob/master/tests/core.py)

Like `from my.core.serialize import test_without_orjson_installed`?

Reply:
Yeah, makes sense! and it could also be called by the orjson version as a normal function